src/main/java/uk/gov/gchq/palisade/service/request/DataRequestConfig.java

```diff
@@ -44,87 +44,86 @@ public DataRequestConfig() {
     }
 
     public DataRequestConfig user(final User user) {
-        requireNonNull(user, "User cannot be set to null.");
-        this.user = user;
+        this.setUser(user);
         return this;
     }
 
-    public void setUser(final User user) {
-        user(user);
+    public DataRequestConfig context(final Context context) {
```

Comment:
this mutator method needs @generated annotation adding

Reply:
Done